index.html

```diff
@@ -264,6 +292,36 @@ <h4 class="sponsor-name">Julia Leda</h4>
           </div>
         </div>
       </section>
+      <section class="partner">
+        <h3 class="partner-title">Partner</h3>
+        <div class="partner-container">
+          <figure>
+            <img src="./images/heart.png" alt="img1" />
+            <figcaption>Neighboor caring</figcaption>
+          </figure>
+          <figure>
+            <img src="./images/solidarity.png" alt="img2" />
+            <figcaption>Charitas Foundation</figcaption>
+          </figure>
+          <figure>
+            <img src="./images/travel.png" alt="" />
+            <figcaption>Travel</figcaption>
+          </figure>
+          <figure>
+            <img src="" alt="" />
+            <figcaption></figcaption>
+          </figure>
+        </div>
+      </section>
+      <footer class="footer">
+        <div>
+          <img src="" alt="" />
+        </div>
+        <div>
+          <p>&copy; 2022 Genuine Charity. Some Rights Reserved.</p>
+          <p>Created by Christian Biringanine</p>
+        </div>
```

Comment:
- [ ] Kindly add the logo in the footer on both pages as requested by the previous reviewer so as to follow the required design 👍 
- [ ] (OPTIONAL) Also, it would be great to add some more padding to the footer so as to closely follow the design 💯 
- Your design:
![Annotation 2022-04-16 080220](https://user-images.githubusercontent.com/85113567/163665781-66961e93-6ca1-44e9-a962-d3c7f777872e.jpg)
- Required design:
![Annotation 2022-04-16 080202](https://user-images.githubusercontent.com/85113567/163665800-d1acf298-126f-4c74-8334-2b9511b267a1.jpg)

Reply:
okay cool!